packages/@office-iss/react-native-win32/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.win32.js

```diff
@@ -39,7 +39,7 @@ type AccessibilityEventTypes = 'click' | 'focus';
 
 // Mapping of public event names to platform-specific event names.
 const EventNames: Map<$Keys<AccessibilityEventDefinitions>, string> =
-  Platform.OS === 'android' || Platform.OS === 'windows'
```

Comment:
I don't remember at this point, but just confirming we do not need this being called in RNW

Reply:
Yea the win32 copy of the file needs `Platform.OS === win32` so the windows check is not needed.